# Message definition for a list of tracked objects

Header header
TrackedObject[] objects
